recipe/meta.yaml

```diff
@@ -1,21 +1,19 @@
 {% set name = "conda-build" %}
-{% set version = "3.21.8" %}
+{% set version = "3.21.9" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/conda/conda-build/releases/download/{{ version }}/conda-build-{{ version }}.tar.gz
-  sha256: 0031aff4a34e66340f90428051d480c6c456c89e267f78cf5987126bee68ca5b
+  url: https://github.com/conda/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 19a5568e09ee476c7e9e68e00583f8ff1e4ca16b709e5078552156194ecd14ee
   patches:
     # backport conda/conda-build#4287
     - patches/0001-Remove-rpaths-that-occur-multiple-times.patch
```

Comment:
Does not apply and can be removed according to https://github.com/conda/conda-build/pull/4287
```suggestion
```